public_html/wp-content/plugins/wordcamp-speaker-feedback/views/view-feedback.php

```diff
@@ -41,6 +44,51 @@
 <div class="speaker-feedback">
 	<h2><?php esc_html_e( 'Session Feedback', 'wordcamporg' ); ?></h2>
 
+	<?php if ( ! $is_session_speaker ) : ?>
+		<p class="speaker-feedback__notice">
+			<?php esc_html_e( 'Only feedback recipients can mark submissions as helpful.', 'wordcamporg' ); ?>
+		</p>
+	<?php endif; ?>
+
+	<?php foreach ( $session_speakers as $session_speaker ) :
+		if ( $is_session_speaker && get_current_user_id() !== $session_speaker ) {
+			continue;
+		}
+		$user = new WP_User( $session_speaker );
+		$notifications_opt_out = get_user_meta( $user->ID, SPEAKER_OPT_OUT_KEY, true );
+		?>
+		<div class="speaker-feedback__notifications <?php echo ( $notifications_opt_out ) ? 'is-disabled' : ''; ?>">
+			<span>
+				<?php if ( ! $is_session_speaker ) : ?>
+					<span class="sft-notifications-speaker-name">
```

Comment:
I know it's long, but these should follow the class name patterns: `speaker-feedback__notifications-speaker-name`, `speaker-feedback__notifications-label-text`, etc. (`sft-` is only used on IDs/names)

Reply:
Fixed with fe3c145